web/templates/email/contact_form.text.eex

```diff
@@ -0,0 +1,9 @@
+Some sent you a message from the Raleigh Design contact form!
+
+Message:
+<%= @body %>
+
+Sent from: <%= @from %>
```

Comment:
Maybe put the `from` above the message since it's short and I think that's the way most people think of emails. I don't feel strongly either way though